src/firmware/boards/robot_stm32h7/io/robot_status.c

```diff
@@ -0,0 +1,58 @@
+#include "firmware/boards/robot_stm32h7/io/robot_status.h"
+
+#include "firmware/boards/robot_stm32h7/io/proto_multicast.h"
+#include "firmware/boards/robot_stm32h7/io/proto_multicast_communication_profile.h"
+#include "shared/proto/robot_status_msg.nanopb.h"
+
+static TbotsProto_RobotStatus g_robot_status_msg;
+
+void io_robot_status_task(void *argument)
+{
+    ProtoMulticastCommunicationProfile_t *comm_profile =
+        (ProtoMulticastCommunicationProfile_t *)argument;
+
+    for (;;)
+    {
```

Comment:
Should there be another TODO here? Seems this isn't updating much of the status?

Reply:
added tickets for every block